web-console/4.0-designs/storage/storage.md

```diff
@@ -29,3 +29,47 @@
 ![form for adding and creating storage](img/create-storage-form.png)
 - Selecting `Create Storage` exposes fields for Storage Class, Name, Access Mode, Size, and the label selector checkbox.
 - Only access modes that are compatible should be available to select in this form.
+
+## Set Default Storage Class
+- Users with appropriate privileges can set a Default Storage Class.
+- THERE ARE TWO WAY USER CAN SET A DEFAULT STORAGE CLASS:
```

Comment:
Should there also be the ability to mark a storage class as the default when it's created?

Reply:
Hi Matt, 
Thanks.
Currently, I have some deliverable in DevConsole. I will update those points and resubmit again soon.
Thanks again

Reply:
> Should there also be the ability to mark a storage class as the default when it's created?

@matthewcarleton, this should be handled in the 'create flow'. My story is focused on changing the 'default'

Reply:
ok that makes sense. Can we do a follow up story that visits that to ensure it doesn't get missed?